app/views/provider_interface/decisions/respond.html.erb

```diff
@@ -3,15 +3,19 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
+      <%= t('page_titles.provider.respond') %>
+    </h1>
+    <%= render ProviderInterface::CourseSummaryComponent.new(application_choice: @application_choice) %>
```

Comment:
Just a question here, shouldn't we wait until the updated make offer is in place or use the new feature flag to control when the component is displayed?

Reply:
On the ticket it says: 
> It should be possible to implement this without waiting for the new Make/Change Offer flow
I also double checked with James for posterity and he confirmed its ok to show the box without a feature flag